etcdserver/api/membership/cluster.go

```diff
@@ -62,7 +62,7 @@ type RaftCluster struct {
 func NewClusterFromURLsMap(lg *zap.Logger, token string, urlsmap types.URLsMap) (*RaftCluster, error) {
 	c := NewCluster(lg, token)
 	for name, urls := range urlsmap {
-		m := NewMember(name, urls, token, nil)
+		m := NewMember(name, urls, token, nil, false) // does not support adding learner via URLsMap
```

Comment:
Add godoc to function and document that learner support is not available in the godoc?

Reply:
Done.